src/components/navBar/navBar.js

```diff
@@ -414,6 +418,7 @@ function MdNavItem($$rAF) {
       'mdNavHref': '@?',
       'mdNavSref': '@?',
       'name': '@',
+      'disabled': '=?ngDisabled',
```

Comment:
I think we should rather observe the `disabled` attribute itself, instead of the `ngDisabled` attribute`.

> `ngDisabled` changes the `disabled` attribute at runtime 

We also do this on the autocomplete and other components, this also allows developers to apply the `disabled` attribute, without having to use the `ng-disabled` attribute.


```html
<md-x-item disabled="disabled">
```